### ISSUE TITLE
Market walkers have very small area, they cover maybe 10 tiles max and are coming back. My granaries and storages were filled with resources, but markets weren't getting them.

### ISSUE THREAD

**Issue by AleCie** [Keriew/augustus#71]
As in my reply at: https://www.reddit.com/r/impressionsgames/comments/gxp1xp/augustus_140_released_caesar_3_with_gameplay/

Attaching save file, augustus.ini and .log
[augustus.zip](https://github.com/Keriew/augustus/files/4750764/augustus.zip)

**Comment by arkimedes1967** [comment]
I think you have at least 2 problems on your map:

- Too much working places and too few people. Turn off some of your industries or mothballet some of them till you reach 0% unemployment.
- The production site of wheat is too far from the graneries. Try to put some **more granaries** near the fields and alter the status from accepting to getting goods on the farest ones. The same logic should be applyed to goods.

Bottom line, and please don't take me wrong, your problem is conceptual not technical.

**Comment by ArtySh0ck** [comment]
actualy, extending of walking range of market ladies would be very good as option in Augustus configuration menu.

**Comment by cdsmith-umn** [comment]
If I recall correctly, they already walk some 40 squares/tiles/whatever. If they're having trouble getting food and other goods to sell, then you need to put granaries and warehouses with those items in them closer, anyway. The farther they have to walk, the longer it's going to take to get stuff to sell, and the more likely it is that your houses will run out of that particular food or good before the market lady can return with it to sell.

With all the different block configurations I've tried over the years, the range of market ladies has never been a problem. My problem always was keeping them in the area I wanted them to be in for selling to houses. Roadblocks have entirely solved that problem now.

**Comment by ArtySh0ck** [comment]
> then perhaps it's time to rethink your block design and/or include more markets.

OR we can have an option in configuration menu about extending of walking range. Everyone who loves C3 sometimes want to have some new experiments with city designs, but sometimes it needs schoolers extedings (already we have this), and market ladies... and i think extending walking range of market ladies is even more important. but, who doesnt like this can turn off this option.

truly, we need this option.

**Comment by subatron** [comment]
> > then perhaps it's time to rethink your block design and/or include more markets.
> 
> OR we can have an option in configuration menu about extending of walking range. Everyone who loves C3 sometimes want to have some new experiments with city designs, but sometimes it needs schoolers extedings (already we have this), and market ladies... and i think extending walking range of market ladies is even more important. but, who doesnt like this can turn off this option.
> 
> truly, we need this option.

Arty... This is literally unnecessary. Check out some of GamerZakh's "City Doctor" videos where he fixes people's cities that have market ladies walking ACROSS THE MAP to get goods. So I'd take the hint from one of the devs, who is being ever so kind as to make sure things work AND making new things for, and build the way the game was intended. They're adding usability or technical improvements (schools range literally didn't make sense so they gave us an option). But a market lady can, and will, walk across the map for a resource. It's not hard to put a warehouse that is set to getting of the resource you need.

**Comment by cdsmith-umn** [comment]
I'm not a developer. If I can find the time to contribute to this project, I definitely will. For the foreseeable future, that won't be the case. My experience in playing is just that it's better to keep resources as close to the markets as possible. My preference is to keep the number of houses served by a market somewhat limited, and this is also affected by block design. I like symmetry, as evidenced by the design I used in the picture above. I tweaked my design since then, dropping one of the market "end caps" from the block. I'm not even sure that two markets are truly necessary in the "end cap"; it's just symmetrical and I should probably drop back to one, move the forum into that spot, and that opens up a spot for some other 2x2 building.

**Comment by crudelios** [comment]
All right, the million dollar question: is this actually a bug that needs addressing or a problem with the op's city design?

**Comment by Keriew** [comment]
Everything seems to be working as designed in the save, closing as not a bug.
